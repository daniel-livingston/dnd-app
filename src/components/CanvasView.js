import React from "react";
import Canvas from "./Canvas";
import View from "./View";

const CanvasView = () => (
	<View
		title='Canvas'
		description='Draw with others in the party to show a map, a scene, or to just have fun.'
	>
		<Canvas />
	</View>
);

export default CanvasView;
