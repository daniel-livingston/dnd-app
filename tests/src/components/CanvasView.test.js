import React from "react";
import { shallow } from "enzyme";
import CanvasView from "../../../src/components/CanvasView";

describe("<CanvasView>", () => {
	test("renders correctly", () => {
		const wrapper = shallow(<CanvasView />);
		expect(wrapper).toMatchSnapshot();
	});
});
