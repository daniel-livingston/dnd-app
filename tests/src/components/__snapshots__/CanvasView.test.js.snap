// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CanvasView> renders correctly 1`] = `
<Connect(View)
  description="Draw with others in the party to show a map, a scene, or to just have fun."
  title="Canvas"
>
  <Connect(Canvas) />
</Connect(View)>
`;
